spec/kitchen_spec/directions/bake_order_headers_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Kitchen::Directions::BakeOrderHeaders do
+  let(:page_with_headers) do
+    book_containing(html:
+      one_chapter_with_one_page_containing(
+        <<~HTML
+          <h3>The top title within the page</h3>
+          <div>Body of introduction
+            <h6>Subtitle</h5>
```

Comment:
There are different tags `<h6>`, `</h5>`. Probably because of that the output is:

![image](https://github.com/openstax/cookbook/assets/40228854/841ceab3-cfe9-4f2b-a812-c97c86bfd8ba)

If I understand correctly it should be `<h3>`, not `<h4>`, right?

Reply:
fixed the subtitle closing tag! i wanted this tag to be an `h6`. this tests that when given `[h3, h6, h5, h6]` we get `[h2, h4, h3, h4]`. while i'm not expecting out of order headers to appear in the actual data, i do want this method to be robust enough to keep all headers at the appropriate level relative to each other if that case does happen to arise.

Reply:
That make sense :+1: Thank you for clarification :relaxed: